vm/interpreter/src/fvm/externs.rs

```diff
@@ -231,3 +222,89 @@ impl<DB: BlockStore> Consensus for ForestExterns<DB> {
         Ok((cf, total_gas))
     }
 }
+
+fn cal_gas_used_from_stats(
```

Comment:
It'd be nice to add some piece of documentation here. I don't get from a quick glimpse what is `r` and `w`, other than poorly named fields. We can't change them, but maybe we can at least explain stuff for those less bright like myself. :)

Reply:
fixed